src/render-html/messageAsHtml.js

```diff
@@ -10,13 +10,16 @@ const briefMessageAsHtml = ({
   isStarred,
   reactions,
   ownEmail,
+  isMentioned,
 }) => `
-  <div class="message message-brief" id="msg-${id}" data-msg-id="${id}">
-    <div class="avatar"><img></div>
-    <div class="content">
-      ${content}
-      ${messageTagsAsHtml(timeEdited, isOutbox, isStarred)}
-      ${messageReactionListAsHtml(reactions, id, ownEmail)}
+  <div
```

Comment:
A better approach is to make this flag-independent.
We should add `data-mentioned="true"`, `data-something` etc.
Then we can style independently with 
```css
[data-mentioned="true"] {
  background: pink;
}
```